packages/web/src/components/gcds-textarea/stories/gcds-textarea.stories.tsx

```diff
@@ -169,8 +185,10 @@ const Template = args =>
   ${args.disabled ? `disabled` : null}
   ${args.value ? `value="${args.value}"` : null}
   ${args.characterCount ? `character-count="${args.characterCount}"` : null}
```

Comment:
The `characterCount` can be removed from the Storybook stories. I couldn't comment on the lines further up (`50`) and down (`358` + `379`) where it needs to be removed as well.